src/main/java/com/onlinebookstore/onlinebookstore/model/User.java

```diff
@@ -0,0 +1,43 @@
+package com.onlinebookstore.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.SQLRestriction;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE users SET isdeleted = true WHERE id=?")
+@SQLRestriction("is_deleted = false")
+public class User {
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Id
+    private Long id;
+
+    @NotBlank
+    @Email

```

Comment:
adding `jakarta.validation.constraints` to entity-model classes does not make much sense because you do not have any `@Valid` used with it anywhere, and such validations are usually done in controllers with DTOs, so let's remove them from the models